### DISCUSSION TITLE
feature request: enable snow as weather

### DISCUSSION THREAD

**Discussion by ghost** [OpenRCT2/OpenRCT2#23154] [Feature requests]
Hi

There is snow (and 2 other types) in the cheats but it never appears in the game.
I would like to request snow as weather in the winter months. It could be cool for people making parks with a snow theme!

**Comment by 733737** [comment]
If snow is added to scenarios it will most likely be added to custom scenarios such as openscenarios only to preserve the original artistic style in playbacks of classic scenarios and to prevent snow in jungle or desert scenarios
